slurm-installation/overview.rst

```diff
@@ -248,3 +246,59 @@ After setting the node up, to bring it back you need to run a Juju *action*:
    juju-compute-GsLk    up   infinite      1   idle juju-01ab62-3
    configurator*     inact   infinite      1   idle juju-01ab62-1
 
+
+CentOS7 Deploys on LXD clouds
+-----------------------------
+
+You need to manually create an LXD image for CentOS7 in order to deploy it with
+Juju.
+
+First step is to download a configuration file describing the image:
+
+.. code-block:: bash
+
+    $ wget https://raw.githubusercontent.com/lxc/lxc-ci/master/images/centos.yaml
+
+Juju needs two extra packages (`sudo` and `openssh-server`) that are not in the
+base image. You need to manually add them in the `packages` section of the yaml
+file. The first `set` of packages in the file should then be:
+
+.. code-block:: bash
+
+    packages:
+      manager: yum
+      update: true
+      cleanup: true
+      sets:
+      - packages:
+        - cronie
+        - cronie-noanacron
+        - curl
+        - dhclient
+        - initscripts
+        - openssh-clients
+        - passwd
+        - policycoreutils
+        - rootfiles
+        - rsyslog
+        - vim-minimal
+        - sudo
+        - openssh-server
+        action: install
+
+Now we need to install `distrobuilder` and generate the image:
+
+.. code-block:: bash
+
+    $ sudo snap install distrobuilder --classic
```

Comment:
Personally, I think prefixing bash commands in markdown with `$` makes it more difficult for developers to copy and paste commands. How do you feel about removing the `$`?

Reply:
i like to prefix with `$` because it makes it easier to distinguish what is the command and what is the output. For example:
```bash
$ foo
bar
```
and 
```bash
foo
bar
```

In the first example, it is clear that `foo` is a command and `bar` is its output. In the second example, we don't know in advance.

Also, all other code blocks on this file have the `$` prefix. I prefer to keep it for consistency.
Thoughts?

Reply:
You make a good point. You have swayed me. I agree with keeping the `$`.